Sources/MagicSDK/Core/Provider/RpcProvider.swift

```diff
@@ -83,8 +83,8 @@ public class RpcProvider: NetworkClient, Web3Provider {
 
 public typealias Web3ResponseCompletion<Result: Codable> = (_ resp: Web3Response<Result>) -> Void
 
-internal extension Web3BytesInitializable {
-    init(_ bytes: Web3BytesRepresentable) throws {
+extension BytesInitializable {
+    init(_ bytes: BytesConvertible ) throws {
```

Comment:
LMK if there was some special reason for the name change here. 👍

Reply:
https://app.shortcut.com/magic-labs/story/36788/rename-bytesinitializable-extension
Yes, it's an old issue that collides with the other packages. Could you leave a note about it?